### PR TITLE
add ignore_index to auc_roc metric

### PR DESCRIPTION
**Pull request by michalozeryflato**

### CODE REVIEW

File: fuse/eval/metrics/libs/classification.py [hunk 1]
```diff
@@ -36,6 +36,7 @@ def auc_roc(
         target: Sequence[Union[np.ndarray, int]],
         sample_weight: Optional[Sequence[Union[np.ndarray, float]]] = None,
         pos_class_index: int = -1,
+        ignore_index: int = None,
```
Comment (mosheraboh) [inline fuse/eval/metrics/libs/classification.py:39]: ignore_index: Optional[int] = None,
Can you please add it in the comments as well?

**Review by mosheraboh — DISMISSED** [review]
Looks good! Indeed it was missing.
Single comment inline.

File: fuse/eval/metrics/libs/classification.py (hunk 1 above)
Comment (michalozeryflato) [reply fuse/eval/metrics/libs/classification.py]: done

**Review by mosheraboh — APPROVED** [review]
Looks great!
Sorry for the long delay 😅

**Pull request merged**